source/unified-test-format/unified-test-format.md

```diff
@@ -523,6 +523,15 @@ The structure of this object is as follows:
         Messages for unspecified components and/or with lower severity levels than those specified MUST be ignored by this
         client's log collector(s) and SHOULD NOT be included in [test.expectLogMessages](#test_expectLogMessages) for this
         client.
+
+    - `observeTracingMessages`: Optional object that configures tracing behavior for the client. The structure of this
+        object is as follows:
+
+        - `enableCommandPayload`: Optional boolean. When set to `true`, enables capturing of command payload details in
+            tracing spans.
+            - If `true`, the test runner SHOULD capture detailed command payload information in tracing spans.
+            - If `false` or omitted, the test runner SHOULD exclude command payload details.
```

Comment:
I realize this impacts the test runner, but should it also refer to configuring the MongoClient to enable tracing?

Reply:
This point is similar to https://github.com/mongodb/specifications/pull/1826#discussion_r2344133832 ? 
This option is both configured via the test runner and should be available to toggle on/off via MongoClient. This should be covered by a prose test 👍